backend/src/config/ssh2-sftp-client/index.ts

```diff
@@ -0,0 +1,30 @@
+import Client from 'ssh2-sftp-client';
+
+const sftp = new Client();
+
+const config = {
+    host: process.env.LAN_FTP_SERVER_HOST || 'your.sftp.server',
+    port: process.env.LAN_FTP_SERVER_ || '22',
+    username: process.env.LAN_FTP_SERVER__USER || "",
+    password: process.env.LAN_FTP_SERVER__PASSWORD || "",
+};
+
+const remotePath = '/remote/path/';
```

Comment:
We should just make this an environment variable

Reply:
ok